server.go

```diff
@@ -31,7 +31,9 @@ func handle(w http.ResponseWriter, r *http.Request) {
 	if upgrade == "websocket" || upgrade == "Websocket" {
 		websocket.Handler(func(ws *websocket.Conn) {
 			//Override default Read/Write timeout with sane value for a web socket request
-			ws.SetDeadline(time.Now().Add(time.Hour * 24))
+			if err := ws.SetDeadline(time.Now().Add(time.Hour * 24)); err != nil {
+				panic(err)
```

Comment:
I think we can handle error gracefully on `server.go`instead of panic. @revel/core thoughts?